WordPress/twentynineteen
# Add a simple build tool to build and watch SASS files

This builds on #163 as an alternative to #138.

Testing Instructions:

- Make sure you have node installed
- `npm install`
- To build CSS from SASS: `npm run build`
- To watch for changes to SASS and rebuild CSS when any occur: `npm run watch`
